[2024-11-18T14:40:34.933+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-18T14:40:34.952+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Ciril_group.create-db manual__2024-11-18T14:40:34.508843+00:00 [queued]>
[2024-11-18T14:40:34.963+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Ciril_group.create-db manual__2024-11-18T14:40:34.508843+00:00 [queued]>
[2024-11-18T14:40:34.964+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-11-18T14:40:34.980+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): create-db> on 2024-11-18 14:40:34.508843+00:00
[2024-11-18T14:40:34.991+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'Ciril_group', 'create-db', 'manual__2024-11-18T14:40:34.508843+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/ciril_group_dag.py', '--cfg-path', '/tmp/tmp7eeu6l2n']
[2024-11-18T14:40:34.997+0000] {standard_task_runner.py:105} INFO - Job 6: Subtask create-db
[2024-11-18T14:40:34.998+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=197) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-18T14:40:34.999+0000] {standard_task_runner.py:72} INFO - Started process 201 to run task
[2024-11-18T14:40:35.047+0000] {task_command.py:467} INFO - Running <TaskInstance: Ciril_group.create-db manual__2024-11-18T14:40:34.508843+00:00 [running]> on host aa723b0e8432
[2024-11-18T14:40:35.127+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Simo' AIRFLOW_CTX_DAG_ID='Ciril_group' AIRFLOW_CTX_TASK_ID='create-db' AIRFLOW_CTX_EXECUTION_DATE='2024-11-18T14:40:34.508843+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-18T14:40:34.508843+00:00'
[2024-11-18T14:40:35.129+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-18T14:40:35.174+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-11-18T14:40:35.175+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/opt/***/create-ciril-db.sh']
[2024-11-18T14:40:35.189+0000] {subprocess.py:86} INFO - Output:
[2024-11-18T14:40:35.235+0000] {subprocess.py:93} INFO - permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Put "http://%2Fvar%2Frun%2Fdocker.sock/v1.24/containers/database/archive?noOverwriteDirNonDir=true&path=%2Ftmp": dial unix /var/run/docker.sock: connect: permission denied
[2024-11-18T14:40:35.259+0000] {subprocess.py:93} INFO - permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Get "http://%2Fvar%2Frun%2Fdocker.sock/v1.24/containers/database/json": dial unix /var/run/docker.sock: connect: permission denied
[2024-11-18T14:40:35.262+0000] {subprocess.py:93} INFO - Error executing SQL script.
[2024-11-18T14:40:35.262+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-11-18T14:40:35.286+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-18T14:40:35.287+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=Ciril_group, task_id=create-db, run_id=manual__2024-11-18T14:40:34.508843+00:00, execution_date=20241118T144034, start_date=20241118T144034, end_date=20241118T144035
[2024-11-18T14:40:35.336+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-18T14:40:35.352+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-18T14:40:35.354+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
